### PR TITLE
fix: remove warnings

### PR DESCRIPTION
**Pull request by tomwim**
### Description
Remove 3 warnings in TTS component

### CODE REVIEW

File: Runtime/IRuntimeConfigurationExtensions.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public static class IRuntimeConfigurationExtensions
         /// <summary>
         /// Return loaded <see cref="TextToSpeechConfiguration"/>.
         /// </summary>
-        public static TextToSpeechConfiguration GetTextToSpeechConfiguration(this IRuntimeConfiguration runtimeConfiguration)
+        public static TextToSpeechConfiguration GetTextToSpeechConfiguration(this BaseRuntimeConfiguration runtimeConfiguration)
```
Comment (aleksei-korolev) [inline Runtime/IRuntimeConfigurationExtensions.cs:18]: API break :(

File: Runtime/Provider/MicrosoftSapiTextToSpeechProvider.cs [hunk 2]
```diff
@@ -82,9 +82,9 @@ public void SetConfig(TextToSpeechConfiguration configuration)
         {
             this.configuration = configuration;
         }
-
+        
         /// <inheritdoc />
-        public async Task<AudioClip> ConvertTextToSpeech(string text)
+        public Task<AudioClip> ConvertTextToSpeech(string text)
```
Comment (aleksei-korolev) [inline Runtime/Provider/MicrosoftSapiTextToSpeechProvider.cs:87]: I don't know if removing async keyword is an API break. @Gusinuhe?

Comment (tomwim) [reply Runtime/Provider/MicrosoftSapiTextToSpeechProvider.cs:87]: image
Does not look like it breaks something at least

Comment (tomwim) [reply Runtime/Provider/MicrosoftSapiTextToSpeechProvider.cs:87]: https://stackoverflow.com/questions/13243975/suppress-warning-cs1998-this-async-method-lacks-await
This also sounds like it is fine

**Comment by tomwim** [comment]
Ready for rereview

**Comment by tomwim** [comment]
@Gusinuhe renamed it and had it in a project with all components and template. did not make issues so renaming it apparently worked without further changes

**Comment by Gusinuhe** [comment]
> @Gusinuhe renamed it and had it in a project with all components and template. did not make issues so renaming it apparently worked without further changes

Yeah, since it is an extension class and is never directly referenced so it should be fine to just rename it.

I am more concerned about not awaiting the `Synthesize` method but I guess it already occurs on a second thread so it should be also safe. I tested it and it seems to work just as before.

**Comment by tomwim** [comment]
> I am more concerned about not awaiting the `Synthesize` method but I guess it already occurs on a second thread so it should be also safe. I tested it and it seems to work just as before.

I tested it too and it worked. Awaiting Synthesize did not do anything I guess because there was no await in Synthesize anyways 🤔 but maybe I just dont get the concept fully

**Comment by Gusinuhe** [comment]
> > I am more concerned about not awaiting the `Synthesize` method but I guess it already occurs on a second thread so it should be also safe. I tested it and it seems to work just as before.
> 
> I tested it too and it worked. Awaiting Synthesize did not do anything I guess because there was no await in Synthesize anyways 🤔 but maybe I just dont get the concept fully

Yeah, you made the right changes 👍

File: Runtime/IRuntimeConfigurationExtensions.cs [hunk 3]
```diff
@@ -1,36 +0,0 @@
-﻿using Innoactive.Creator.Core.Configuration;
-
-namespace Innoactive.Creator.TextToSpeech
-{
-    /// <summary>
-    /// TextToSpeech extensions methods for <see cref="IRuntimeConfiguration"/>.
-    /// </summary>
-    public static class IRuntimeConfigurationExtensions
```
Comment (aleksei-korolev) [inline Runtime/IRuntimeConfigurationExtensions.cs:8]: API break. I suggest keeping this class alongside the new one, but mark it as obsolete.

Comment (aleksei-korolev) [reply Runtime/IRuntimeConfigurationExtensions.cs:8]: (This is an API break because you can call these methods just like normal static methods)

**Comment by tomwim** [comment]
Ready for re-review

**Pull request merged**